server/resource_server/service/service_test.go

```diff
@@ -65,9 +70,9 @@ type mockRepository struct {
 	findUser                     func(userId uint) (models.User, error)
 	saveEmailVerificationData    func(data models.EmailVerificationData) error
 	getEmailVerificationData     func(userId uint) (models.EmailVerificationData, error)
-	deleteEmailVerificationData  func(userId uint) error
+	// deleteEmailVerificationData  func(userId uint) error
```

Comment:
You can remove `deleteEmailVerificationData` and `setUserEmailVerified` fields as they are not used (rather then just commenting them out)